# TOPMed RNA-seq paper index repo

This is the top-level repo for the TOPMed RNA-seq paper.

# Pre-scan processing

* [Code displaying the RNA-seq mapping process is here](https://github.com/broadinstitute/gtex-pipeline/blob/aca6bccc3c263a6afdc2993c8c066af927b7bbd0/TOPMed_RNAseq_pipeline.md)
* [Code displaying the RNA-seq QC procedure is here](https://github.com/porchard/topmed-cohort-level-qc)
* [Code displaying the selection of unrelated samples is here](https://github.com/porchard/topmed-process-rnaseq-sample-genotypes)
* [Mappability scores were generated using this code](https://github.com/porchard/crossmap-nextflow)
* [Code displaying RNA-seq variant-sensitive (WASP) mapping process is here](https://github.com/porchard/topmed-rnaseq-remapping-and-leafcutter)
* [Code to generate e/sQTL scan input is here](https://github.com/porchard/topmed-qtl-input)
  
# Scans
* [Code to run scans is here](https://github.com/porchard/topmed-qtl-scans)

# Processing of external data
* [Pre-process eQTLGen data](https://github.com/porchard/topmed-eqtlgen-preprocessing)
* [Finemap PanUKBB GWAS](https://github.com/porchard/panukbb-finemapping)
* [Pre-process GTEx data and coloc w/ PanUKBB GWAS](https://github.com/porchard/topmed-gtex-preprocessing)


# Post-scan processing

* [Run motif scans](https://github.com/porchard/topmed-motif-scanning)
* [Run other intermediate processing steps](https://github.com/porchard/topmed-intermediate-processing)

# Figures
* [Code for generating figures is here](https://github.com/porchard/topmed-rnaseq-paper-figures)